apidoc/Titanium/UI/Snackbar.yml

```diff
@@ -0,0 +1,82 @@
+---
+name: Titanium.UI.Snackbar
+summary: Snackbars provide brief messages about app processes at the bottom of the screen.
+description: |
+    For design guidelines, see
+    [Material design: Snackbars](https://material.io/components/snackbars)
+since: "10.2.0"
+platforms: [android, iphone, ipad, macos]
```

Comment:
Move to `Ti.UI.Android.*` namespace for now and remove the other platforms, until we have a proper parity API

Reply:
moved everything to the Android namespace